opentelemetry-sdk/src/opentelemetry/sdk/metrics/_internal/aggregation.py

```diff
@@ -349,6 +362,329 @@ def collect(
+                return None
+
+            start_time_unix_nano = self._start_time_unix_nano
+            sum_ = self._sum
+            max_ = self._max
+            if max_ == -inf:
+                max_ = None
+            min_ = self._min
+            if min_ == inf:
+                min_ = None
+
+        if self._count == self._zero_count:
+            scale = 0
+
+        else:
+            scale = self._mapping.scale
+
+        current_point = ExponentialHistogramDataPoint(
+            attributes=self._attributes,
+            start_time_unix_nano=start_time_unix_nano,
+            time_unix_nano=collection_start_nano,
+            count=self._count,
+            sum=sum_,
+            scale=scale,
+            zero_count=self._zero_count,
+            positive=BucketsPoint(
+                offset=self._positive.offset,
+                bucket_counts=self._positive.counts,
+            ),
+            negative=BucketsPoint(
+                offset=self._negative.offset,
+                bucket_counts=self._negative.counts,
+            ),
+            # FIXME: Find the right value for flags
+            flags=0,
+            min=min_,
+            max=max_,
+        )
+
+        if self._previous_point is None or (
+            self._instrument_temporality is aggregation_temporality
+        ):
+            self._previous_point = current_point
+            return current_point
+
+        max_ = current_point.max
+        min_ = current_point.min
+
+        if aggregation_temporality is AggregationTemporality.CUMULATIVE:
+            start_time_unix_nano = self._previous_point.start_time_unix_nano
+            sum_ = current_point.sum + self._previous_point.sum
+            # Only update min/max on delta -> cumulative
+            max_ = max(current_point.max, self._previous_point.max)
+            min_ = min(current_point.min, self._previous_point.min)
+
+            negative_counts = [
+                curr_count + prev_count
+                for curr_count, prev_count in zip(
+                    current_point.negative.bucket_counts,
+                    self._previous_point.negative.bucket_counts,
+                )
+            ]
+            positive_counts = [
+                curr_count + prev_count
+                for curr_count, prev_count in zip(
+                    current_point.positive.bucket_counts,
+                    self._previous_point.positive.bucket_counts,
+                )
+            ]
+        else:
+            start_time_unix_nano = self._previous_point.time_unix_nano
+            sum_ = current_point.sum - self._previous_point.sum
+
+            negative_counts = [
+                curr_count + prev_count
+                for curr_count, prev_count in zip(
+                    current_point.negative.bucket_counts,
+                    self._previous_point.negative.bucket_counts,
+                )
+            ]
+            positive_counts = [
+                curr_count + prev_count
```

Comment:
Shouldn't this be `curr_count - prev_count `

Reply:
Correct, fixed